libdokan/rekey_file.go

```diff
@@ -24,7 +24,9 @@ func (f *RekeyFile) WriteFile(ctx context.Context, fi *dokan.FileInfo, bs []byte
 	if len(bs) == 0 {
 		return 0, nil
 	}
-	err = f.folder.fs.config.KBFSOps().Rekey(ctx, f.folder.getFolderBranch().Tlf)
+	// TODO: get rid of this and use RequestRekey instead
```

Comment:
The difference between this and `RequestRekey` is, this is sync, and `RequestRekey` is async. We really only wait for rekey to finish here other than in tests. I was thinking we could get rid of "waiting for rekeys" in non-tests altogether, since a return from `rekeyLocked` doesn't necessarily mean the rekey is finished. It can spawn paper rekey prompts, or regular 30s rekey rechecks, which are all caused by the same rekey request.